server/service/src/sync/program_schemas/encounters_report.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
:nit

the orange borders don't do it for me..

<img width="397" alt="image" src="https://user-images.githubusercontent.com/9192912/218342921-7cdfc526-8254-4129-9f15-ea43acc7db22.png">

it's an example though, right? So no need to change. I'm just being whiney.

Reply:
Yes its just about getting all the data right now! I still need to do the style it.